src/cli_plugin/cli.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import pkg from '../utils/packageJson';
+import Command from '../cli/Command';
+
+let argv = process.env.kbnWorkerArgv ? JSON.parse(process.env.kbnWorkerArgv) : process.argv.slice();
+let program = new Command('bin/kibana-plugin');
+
+program
+.version(pkg.version)
+.description(
+  'Kibana is an open source (Apache Licensed), browser ' +
+  'based analytics and search dashboard for Elasticsearch.'
+);
+
+require('./list')(program);
```

Comment:
I believe with the move to ES6 modules, we've gotten rid of require statements on client and server code now. If I'm right, you should be using import instead of require here, and elsewhere.